packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash/sqm-tax-and-cash.tsx

```diff
@@ -0,0 +1,1041 @@
+   */
+  @Prop() step2_selectedRegion: string = "Country / Region of Indirect Tax";
+  /**
+   * Label text for the Province select input
+   * @uiName Province select input label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_province: string = "Province";
+
+  /**
+   * Label text for the Indirect Tax Number input
+   * @uiName Indirect Tax Number input label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_indirectTaxNumber: string =
+    "{taxType, select, GST {GST Number} HST {HST Number} VAT {VAT Number} CT {CT Number} SST {SST Number} GENERAL {Indirect Tax Number}}";
+  /**
+   * Label text for the QST Tax Number input
+   * @uiName QST Tax Number input label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_qstNumber: string = "QST Number";
+  /**
+   * Label text for the QST checkbox
+   * @uiName checkbox label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_isRegisteredQST: string = "I am registered for QST Tax";
+  /**
+   * Label text for Income Tax checkbox
+   * @uiName Income Tax checkbox label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_isRegisteredSubRegionIncomeTax: string =
+    "I am an individual registered for Income Tax purposes in Spain, and withholding tax will apply to any payments made to me.";
+  /**
+   * Label text for Sub-region select
+   * @uiName for Sub-region select label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_subRegion: string = "Sub-region";
+  /**
+   * Label text for the Sub-region Income Tax Number input
+   * @uiName Sub-region income Tax Number input label
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_subRegionTaxNumberLabel: string = "Income Tax Number";
+  /**
+   * Alert header text shown in alert if user is already a registered partner
+   * @uiName Participant is partner title
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_isPartnerAlertHeader: string =
+    "An account with this email already exists with our referral program provider, impact.com";
+
+  /**
+   * Alert description text shown in alert if user is already a registered partner
+   * @uiName Participant is partner description
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_isPartnerAlertDescription: string =
+    "If you don’t recognize this referral program provider or believe this is a mistake, please contact Support or sign up for this referral program with a different email.";
+
+  /**
+   * Text shown inside of submit button
+   * @uiName Submit button text
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_submitButton: string = "Continue";
+  /**
+   * Text shown inside of back button
+   * @uiName Back button text
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_backButton: string = "Back";
+  /**
+   * Error text shown below the tax details radio buttons
+   * @uiName Indirect tax details error text
+   * @uiGroup Step 2 Properties
+   */
+  @Prop() step2_taxDetailsError: string = "This field is required";
```

Comment:
We already have a prop with the same value, we could just condense the two into a global prop for the form